packages/Babylonian-Printbugger.package/BPPrintbuggerContext.class/instance/track..st

```diff
@@ -0,0 +1,7 @@
+state
+track: aBPExample
```

Comment:
What does track mean for this class? :) For examples it seems to mean that we observe them, for TraceValues it seems to mean something else. Maybe use different vocabulary for the two cases.